main-site/components/adventure-ahead-section/AdventureAheadSection.styles.ts

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+import { P } from '../../../shared-ui/style/typography';
+import { min, max } from '../../../shared-ui/lib/responsive';
+import { colors } from '../../../shared-ui/style/colors';
+
+const StyledAdventureAheadSectionContainer = styled.div`
+  padding-top: 16em;
+
+  @media ${min.mobile} {
+    margin-top: 14em;
+  }
+  @media ${min.tabletSm} {
+    margin-top: 24em;
+  }
+  @media ${min.tablet} {
+    margin-top: 4em;
+  }
+  @media ${min.tabletLg} {
+    margin-top: 10em;
+  }
+  @media ${min.desktop} {
+    margin-top: 23em;
```

Comment:
Do you need this much margin / padding for each screensize? I think you can just simplify all of this to something like margin: 10em 0; to add some top + bottom margin.